packages/module-bbs/tests/util.ts

```diff
@@ -0,0 +1,19 @@
+export function testSkipNode17And18(...parameters: Parameters<typeof test>) {
```

Comment:
Will this prompt an output as to why they are being skipped?

Reply:
Nope... :)

Reply:
Do you know if this is something we can add? Maybe just a `console.warn` giving a small explanation and linking to an issue?